src/utils/compare-events/index.test.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable caleb/jest/expect-expect */
```

Comment:
I think you can undo this change? Not sure what it's for 😕

Reply:
There were a lot of issues being thrown about tests that didn't work for some reason. I can take it out if you don't want it, but I think it'll make the lint cleaner in the future.

Reply:
Oh I see what this is flagging. Can you make a separate PR to add it to the project eslint config instead of just this individual file? https://github.com/Pigmice2733/peregrine-frontend/blob/dev/package.json#L118